cli/src/main/java/io/aklivity/zillabase/cli/internal/commands/start/ZillabaseStartCommand.java

```diff
@@ -717,6 +779,31 @@ else if (!operations.isEmpty())
                 southKafkaClient.kind = "client";
                 optionsConfig = new ZillaBindingOptionsConfig();
                 optionsConfig.specs = Map.of("kafka_api", kafkaApi);
+                List<ZillaBindingOptionsConfig.KafkaTopicConfig> topicsConfig = new ArrayList<>();
+                for (String reply : replies)
+                {
+                    ZillaBindingOptionsConfig.KafkaTopicConfig topicConfig =
+                        new ZillaBindingOptionsConfig.KafkaTopicConfig();
+                    topicConfig.name = "%s.%s".formatted(config.risingwave.db, reply);
+                    ZillaBindingOptionsConfig.TransformConfig transforms =
+                        new ZillaBindingOptionsConfig.TransformConfig();
+                    transforms.headers = Map.of(":status", "${message.value.status}",
+                        "zilla:correlation-id", "${message.value.correlation_id}");
+                    ZillaBindingOptionsConfig.ModelConfig value = new ZillaBindingOptionsConfig.ModelConfig();
+                    value.view = "json";
+                    value.model = "avro";
+                    value.catalog = Map.of("catalog0",
+                        List.of(Map.of("subject", "%s.%s-value".formatted(config.risingwave.db, reply),
+                            "version", "latest")));
+                    topicConfig.value = value;
+                    topicConfig.transforms = List.of(transforms);
+                    topicsConfig.add(topicConfig);
+                }
```

Comment:
The `subject` is based on the topic name with `-value` suffix, so we are re-deriving the fully qualified kafka topic name here, based on the channel name, prefixing with `db.`.

However, I think that is already the reply channel address, right?
If so, we should be keeping the reply channel address rather than the reply channel name, then no need to re-derive the channel address (topic name) again to construct the subject, agree?